pkg/pathutils/pathutils.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright © 2021 Aspect Build Systems Inc
+Not licensed for re-use.
+*/
+
+package pathutils
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+// https://github.com/bazelbuild/bazel/blob/8346ea4c/src/main/cpp/workspace_layout.cc#L37
+var workspaceFilenames = []string{"WORKSPACE", "WORKSPACE.bazel"}
+
+type RunFn func(cmd *cobra.Command, args []string) (exitErr error)
+
+func InvokeCmdInsideWorkspace(toRun RunFn) (wrapped RunFn) {
```

Comment:
```suggestion
// InvokeCmdInsideWorkspace verifies if the current working directory is inside a Bazel workspace, then invokes the provided toRun function.
func InvokeCmdInsideWorkspace(toRun RunFn) RunFn {
```

Reply:
I also removed the named return argument `wrapped` since we don't use it as a named arg.